pkg/vlan/integration_test.go

```diff
@@ -41,10 +41,11 @@ var _ = Describe("vlan client", func() {
 			summary, err := api.Create(ctx, def)
 			Expect(err).NotTo(HaveOccurred())
 			vlanID = summary.Identifier
-		})
-		DeferCleanup(func(ctx context.Context) {
-			err := api.Delete(ctx, vlanID)
-			Expect(err).NotTo(HaveOccurred())
+
+			DeferCleanup(func(ctx context.Context) {
+				err := api.Delete(ctx, vlanID)
+				Expect(err).NotTo(HaveOccurred())
+			})
```

Comment:
Same goes for this change, please create a new PR for it.

Reply:
This is part of this PR as it only affects/fixes integration tests.